pom.xml

```diff
@@ -3,7 +3,7 @@
     <groupId>org.littleshoot</groupId>
     <artifactId>littleproxy</artifactId>
     <packaging>jar</packaging>
-    <version>1.1.3.2-VGS-SNAPSHOT</version>
+    <version>1.1.3.3-VGS-SNAPSHOT</version>
```

Comment:
we should be respecting sem ver and bumping the minor build number when we make breaking changes. i think we missed this on the last version.